### DISCUSSION TITLE
Web Panel white screen only

### DISCUSSION THREAD

**Discussion by kimpoy010** [ant-media/Ant-Media-Server#6354] [Q&A]
I just finished setting up ant media on a VPS by following [this ](https://antmedia.io/docs/guides/installing-on-linux/installing-ams-on-linux/#automatic-download-and-install-ant-media-server-ams) guide.

I enabled all necessary ports in my firewall but the web panel isn't showing the account creation page. It only shows a blank page.

![2024-05-22_23h37_21](https://github.com/ant-media/Ant-Media-Server/assets/14977470/d5ef87bc-556c-48c0-b307-770007dd1b12)

**Comment by Mohit-3196** [comment]
Hi,
It seems to be related to resource type.
May I know the server specifications on which you have installed the AMS?

Thank you,
Mohit

**Reply by yashtandon113** [reply]
@kimpoy010 
When I tried to access your server through your domain, it appeared to work properly. Please try incognito mode or clear your browser cache.

![image](https://github.com/ant-media/Ant-Media-Server/assets/82374739/106efa6c-d176-413a-ae2c-93e03a511798)
